src/api/app/jobs/project/update_packages_if_dirty.rb

```diff
@@ -0,0 +1,7 @@
+class Project
```

Comment:
For me is better to not use the same Namespace (in this case is the Project class)